### ISSUE TITLE
Don't suppress unhandled rejections

### ISSUE THREAD

**Issue by benjamingr** [mattlewis92/angular-bluebird-promises#3]
>       Bluebird.onPossiblyUnhandledRejection(angular.noop);

This strips a lot of the motivation and reason to use bluebird in the first place.

Also note - that you could use `Promise.props` for your `all` shim.

**Comment by mattlewis92** [comment]
My reasoning behind that was that I wanted this to be a drop in replacement for $q, that included not logging warnings by default for uncaught errors. When used with the ui-router for example lots of messages were logged to the console by default. However it is easy to override the default bluebird handler in your own code like so:
```javascript
angular.module('mwl.bluebird').run(function($q, $log) {
  $q.onPossiblyUnhandledRejection(function(err) {
    $log.warn('Unhandled rejection', err);
  });
});
```

I've added a note to the docs making this more clear and encouraging developers to do so. 

And I didn't know about `Promise.props`, I've updated the code to use that now.

Thanks for the feedback! :)
